✔ |  OK F W S | Context

⠏ |   0       | backtraces                                                      
⠋ |   0 1     | backtraces                                                      
⠙ |   0 2     | backtraces                                                      
⠹ |   0 3     | backtraces                                                      
⠸ |   0 4     | backtraces                                                      
⠼ |   0 5     | backtraces                                                      
⠴ |   0 6     | backtraces                                                      
⠦ |   0 6 1   | backtraces                                                      
⠧ |   1 6 1   | backtraces                                                      
✖ |   1 6 1   | backtraces
────────────────────────────────────────────────────────────────────────────────
backtraces.R:6:3: error: errors thrown at block level are entraced
Error: foo
Backtrace:
 1. f()
 2. g()

backtraces.R:11:3: error: errors thrown from a quasi-labelled argument are entraced
Error: foo
Backtrace:
 1. testthat::expect_s3_class(foo(), "foo")
 4. foo()

backtraces.R:18:3: error: errors thrown from a quasi-labelled argument are entraced (deep case)
Error: foo
Backtrace:
 1. testthat::expect_s3_class(f(), "foo")
 4. f()
 5. g()
 9. foo()

backtraces.R:28:3: error: errors thrown from a quasi-labelled argument are entraced (deep deep case)
Error: foobar
Backtrace:
 1. f()
 2. g()
 6. foo()
 7. bar()

backtraces.R:35:3: error: failed expect_error() prints a backtrace
Error: bar
Backtrace:
 1. testthat::expect_error(f(), "foo")
 7. f()
 8. signaller()

backtraces.R:43:3: error: Errors are inspected with `conditionMessage()`
Error: dispatched

backtraces.R:50:3: warning: also get backtraces for warnings
foobar
Backtrace:
 1. foo()
 2. bar()
────────────────────────────────────────────────────────────────────────────────

══ Results ═════════════════════════════════════════════════════════════════════
OK:       1
Failed:   6
Warnings: 1
Skipped:  0
